pkg/eks/components/CruEKS.vue

```diff
@@ -409,6 +418,9 @@ export default defineComponent({
     },
 
     async actuallySave(): Promise<void> {
+      if (!this.isNewOrUnprovisioned && !this.nodeGroups.length) {
+        delete this.normanCluster.eksConfig.nodeGroups;
```

Comment:
`this.normanCluster.eksConfig` is possibly undefined. We can include it in the if condition before delete.